ServiceConfig.ts

```diff
@@ -1,13 +1,20 @@
 import 'dotenv/config';
-import { AccessKey, createFromBase64EncodedAccessKey} from '@laserfiche/lf-api-client-core';
-export const servicePrincipalKey: string =
-  process.env.SERVICE_PRINCIPAL_KEY ?? '';
+import { authorizationTypeEnum } from './AuthorizationType.js';
+import { AccessKey, createFromBase64EncodedAccessKey } from '@laserfiche/lf-api-client-core';
+export const servicePrincipalKey: string = process.env.SERVICE_PRINCIPAL_KEY ?? '';
 if (!servicePrincipalKey) {
   throw new Error(`Unable to load SERVICE_PRINCIPAL_KEY from .env`);
```

Comment:
I followed the instructions in the README.md for selfhosted and get an error thrown from this line since my `.env` only contains the 5 variables and doesn't contain `SERVICE_PRINCIPAL_KEY`. This shouldn't be a required environment variable anymore?
https://github.com/Laserfiche/lf-sample-repository-api-nodejs#self-hosted-prerequisites